src/lib/cimb_translator/CimbReader.cpp

```diff
@@ -27,18 +27,20 @@ namespace {
 	template <typename MAT>
 	bitbuffer preprocessSymbolGrid(const MAT& img, bool needs_sharpen)
 	{
+		// will this need to change for smaller tiles? probably?
+
 		int blockSize = 3; // default: no preprocessing
 
 		cv::Mat symbols;
 		cv::cvtColor(img, symbols, cv::COLOR_RGB2GRAY);
 		if (needs_sharpen)
 		{
 			sharpenSymbolGrid(symbols, symbols);
-			blockSize = 7;
+			blockSize = 3;
 		}
-		cv::adaptiveThreshold(symbols, symbols, 255, cv::ADAPTIVE_THRESH_MEAN_C, cv::THRESH_BINARY, blockSize, 0);
+		cv::adaptiveThreshold(symbols, symbols, 255, cv::ADAPTIVE_THRESH_MEAN_C, cv::THRESH_BINARY, blockSize, -5);
```

Comment:
The magic formula is grayscale algorithm (e.g. what RGB formula we use -- happens to be opencv's default) + blocksize + the constant. `(3,-5)` is my current best guess, but it'll be prone to some minor overfitting.